services/app-web/src/pages/StateSubmission/StateSubmissionForm.tsx

```diff
@@ -289,7 +293,13 @@ export const StateSubmissionForm = (): React.ReactElement => {
                         />
                     </Route>
                     <Route path={RoutesRecord.SUBMISSIONS_REVIEW_SUBMIT}>
-                        <ReviewSubmit draftSubmission={draft} unlocked={!!unlockedInfo} />
+                        <ReviewSubmit
+                            draftSubmission={draft}
+                            unlocked={!!unlockedInfo}
+                        />
+                    </Route>
+                    <Route path='*'>
+                        <Error404 />
```

Comment:
This is to address https://qmacbis.atlassian.net/browse/OY2-10054 which I saw in the backlog this morning.